src/PantherTestCaseTrait.php

```diff
@@ -159,7 +159,10 @@ protected static function createPantherClient(array $options = [], array $kernel
         }
 
         if (\is_callable([self::class, 'getClient'])) {
```

Comment:
Can be merged in a single if: `if (\is_callable([self::class, 'getClient']) && (new \ReflectionMethod(self::class, 'getClient'))->isStatic()) {`

Reply:
Alright. See next commit.